sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py

```diff
@@ -82,8 +82,8 @@
     install_requires=[
         "azure-core<2.0.0,>=1.23.0",
         "msrest>=0.6.10",
-        "opentelemetry-api<2.0.0,>=1.12.0rc1,!=1.10a0",
-        "opentelemetry-sdk<2.0.0,>=1.12.0rc1,!=1.10a0",
```

Comment:
Just curious, what did the <2.0.0,> mean?

Reply:
It means we don't want to install 2.* versions of the package (because there might be breaking changes). See https://semver.org/#why-use-semantic-versioning